src/net_processing.cpp

```diff
@@ -69,6 +69,8 @@ static constexpr int HISTORICAL_BLOCK_AGE = 7 * 24 * 60 * 60;
 static constexpr int32_t MAX_PEER_TX_IN_FLIGHT = 100;
 /** Maximum number of announced transactions from a peer */
 static constexpr int32_t MAX_PEER_TX_ANNOUNCEMENTS = 2 * MAX_INV_SZ;
+/** How many microseconds to delay requesting transactions via txids, if we have wtxid-relaying peers */
```

Comment:
Because of this confusion I am generally not a fan of mentioning the type in the comment. I don't see why it's useful since ``std::chrono::microseconds`` definition is clear enough.

Reply:
I see @naumenkogs's point that this is redundant, but I don't understand why @jonatack is saying this is incorrect?  TXID_RELAY_DELAY is the number of microseconds that we wait before requesting a transaction, no?

Reply:
The value in the definition is expressed in seconds (albeit with microsecond precision), e.g. a delay of 2 seconds and not 2 microseconds, unless I am confused. There are several similar cases; I think this can be ignored and addressed (or not) separately.

Reply:
The comment is technically correct because it's referring to the `TXID_RELAY_DELAY` constant, not how it is computed. Compare it with how before it might have been `static const int TXID_RELAY_DELAY = 2 * 1000000`, where 2 was still a value in seconds.

It's a silly semantics discussion though. With the std::chrono types everywhere the comment could just be "Duration of delay ...".

Reply:
> the comment could just be "Duration of delay ...".

Yes.